clients/java/src/main/java/io/camunda/client/api/search/filter/ProcessDefinitionFilter.java

```diff
@@ -24,7 +24,7 @@ public interface ProcessDefinitionFilter extends SearchRequestFilter {
    *
    * @return the updated filter
    */
-  ProcessDefinitionFilter latest();
+  ProcessDefinitionFilter isLatestVersion();
```

Comment:
🔧 For consistency with other client filters, we should let users hand in a boolean parameter here. That's how other filters like `ProcessInstanceFilter::hasIncident` work.